05/DB.test.js

```diff
@@ -1 +1,75 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+it("check if id is a number", async () => {
+  expect.assertions(1);
+  const db = new DB();
+  const id = "string";
+
+  try {
+    await db.insert({ id: id });
+  } catch (err) {
+    expect(err).toBe("ID can be only number!");
+  }
+});
+
+it("check if id is not duplicated", async () => {
+  const db = new DB();
+  const rowsLength = db.getRows().length;
+  expect.assertions(rowsLength);

```

Comment:
Tutaj nie powinniśmy wstawiać zmiennych - może to powodować trudne do zidentyfikowania błędu.

Powinno być `expect.assertions(1);`, natomiast dalej możemy to sprawdzić tak:
```
try {
    await db.insert({ id: id });
     await db.insert({ id: id });
  } catch (err) {
    expect(err).toBe("ID can't be duplicated!");
  }
```